Makefile

```diff
@@ -213,21 +232,10 @@ dist/profiles/$(PROFILE)/native/%.o: $(PROFILE_DIR)/native/%.c \
 test-build: $(LIBC_SO) \
             $(LIBSTDCXX_SO) \
             $(call timestamp_of,c-cpp)
-	@echo "Testing kcc..."
```

Comment:
I've made this an independent target, `simple-build-test`. Given the array of tests we run in the CI, this seems like it slows down (re)compilation unnecessarily.